src/index.html

```diff
@@ -17,16 +17,40 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <search class="search">
+      <div
+        class="search__large"
+        data-qa="big"
+      >
+        <img
+          class="search__img search__img--large"
+          src="./images/Search.svg"
+          alt="search"
+        />
+        <input
+          class="search__input search__input--large"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </div>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <div
+        class="search__small"
+        data-qa="small"
+      >
+        <img
+          class="search__img search__img--small"
+          src="./images/Search.svg"
+          alt="search"
+        />
+        <input
+          class="search__input search__input--small"
+          type="text"
+          data-qa="hover"
+          placeholder="Try “Los Angeles“"
+        />
+      </div>
+    </search>
```

Comment:
The `<search>` tag is not a standard HTML element. Consider using a `<div>` or `<form>` element instead, depending on the intended functionality of this section. If this is meant to be a search form, using a `<form>` element would be more semantically correct.